Source/SwiftLintFramework/Rules/Lint/PrivateUnitTestRule.swift

```diff
@@ -63,6 +73,12 @@ public struct PrivateUnitTestRule: ASTRule, ConfigurationProviderRule, CacheDesc
                 "internal func test2() {}\n " +
                 "public func test3() {}\n " +
             "}",
+            "@objc" +
+            "private class FooTest: XCTestCase { " +
```

Comment:
This will generate "@objcprivate". How about keeping it in one line as "@objc private class ..."